packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1517,6 +1517,19 @@ class Positioned extends ParentDataWidget<Stack> {
        bottom = null,
        super(key: key, child: child);
 
+  /// Creates a Positioned object with [left], [top], [right], and [bottom] set
+  /// to 0.0.
+  Positioned.fixedToEdge({
```

Comment:
This seems like a useful shorthand but maybe not the best name. How about Positioned.toEdges()?  Or you could define Positioned.fromInsets({ EdgeInsets insets }), like Positioned.fromRect(), and make the default for insets be EdgeInsets.zero.